tool/src/main.rs

```diff
@@ -29,6 +29,9 @@ struct Opt {
     #[structopt(short, long, parse(from_os_str))]
     docs: Option<PathBuf>,
 
+    #[structopt(short = "-u", long)]
+    docs_base_urls: Vec<String>,
```

Comment:
if we follow my suggestion above this can just be a single URL, defaults to docs.rs